source/auth/auth.rst

```diff
@@ -171,6 +171,10 @@ handshake:
 #. If the server is of type RSArbiter, no authentication is possible and the
    handshake is complete.
 
+#. Inspect the value of ``maxWireVersion``. If the value is greater than or
+   equal to ``6``, then the driver MUST use ``OP_MSG`` for authentication.
+   Otherwise, it MUST use ``OP_QUERY``.
+
```

Comment:
Could you update the "Spec Version" to 1.10.5, bump the "last modified date", and add a changelog entry at the bottom?

Reply:
Done.